tilt_devnet/docker_images/Dockerfile.ethereum

```diff
@@ -1,39 +1,17 @@
-# syntax=docker.io/docker/dockerfile:1.3@sha256:42399d4635eddd7a9b8a24be879d2f9a930d0ed040a61324cfdf59ef1357b3b2
-FROM node:lts-alpine@sha256:2ae9624a39ce437e7f58931a5747fdc60224c6e40f8980db90728de58e22af7c
+# Defined in tilt_devnet/docker_images/Dockerfile.lerna
+FROM lerna
 
-# npm wants to clone random Git repositories - lovely.
-RUN apk add git python make build-base
+USER root
+RUN apt-get update && apt-get install -y ncat
 
-# Run as user, otherwise, npx explodes.
+# Run as node, otherwise, npx explodes.
 USER 1000
-RUN mkdir -p /home/node/ethereum
-RUN mkdir -p /home/node/.npm
-
-WORKDIR /home/node/governance/xc_governance_sdk_js/
-ADD --chown=node:node governance/xc_governance_sdk_js/ .
-
-RUN --mount=type=cache,uid=1000,gid=1000,target=/home/node/.npm \
-  --mount=type=cache,uid=1000,gid=1000,target=xc_gc_node_modules \
-  npm install && \
-  cp -r node_modules node_modules_cache
+WORKDIR /home/node
+COPY --chown=1000:1000 governance/multisig_wh_message_builder governance/multisig_wh_message_builder
+COPY --chown=1000:1000 governance/xc_governance_sdk_js governance/xc_governance_sdk_js
+COPY --chown=1000:1000 target_chains/ethereum target_chains/ethereum
```

Comment:
I looked a little bit and couldn't find a way. Maybe there be no problem without changing owners but it can cause unexpected behaviors in the future.